api/base/exceptions.py

```diff
@@ -1,5 +1,36 @@
-from rest_framework.exceptions import APIException
+
 from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+def json_api_exception_handler(exc, context):
+    """ Custom exception handler that returns errors object as an array """
+
+    # Import inside method to avoid errors when the OSF is loaded without Django
+    from rest_framework.views import exception_handler
+    response = exception_handler(exc, context)
+
+    # Error objects may have the following members. Title removed to avoid clash with node "title" errors.
```

Comment:
If you try to create a node without supplying a title, it will return:
![screen shot 2015-09-01 at 4 41 04 pm](https://cloud.githubusercontent.com/assets/9755598/9615691/d79af12e-50c7-11e5-98c9-53f8a27341a8.png)

The field name happens to be `title` , which is also the name of one of the acceptable top-level members.    But what we want to communicate here is that the title is blank.  Not use `title` as a top level field here.  I would want `title` being blank to be formatted the same as `category` being blank, if it was required.